.vscode/launch.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Better without this file

Reply:
I find it useful to have in all our repos, but can remove it of course.